packages/server/tests/integration/server.spec.ts

```diff
@@ -1111,7 +1111,38 @@ describe('RPC Server', async function() {
 
           Assertions.expectedError();
         } catch (error) {
-          BaseTest.invalidParamError(error.response, Validator.ERROR_CODE, `Invalid parameter 1: ${Validator.BLOCK_NUMBER_ERROR}`);
+          BaseTest.invalidParamError(error.response, Validator.ERROR_CODE, `Invalid parameter 1: ${Validator.BLOCK_PARAMS_ERROR}`);
+        }
+      });
+
+      it('validates Block param is valid block hash', async function() {
```

Comment:
These should be labelled "non valid" right?
The current logic expects failures

Reply:
Yep, you are correct. Fixed.